ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -429,7 +429,10 @@ objects:
         fi
         if [[ "${CLUSTER_TYPE}" = "aws" ]]; then
             master_type=null
-            if [[ "${CLUSTER_VARIANT}" =~ "xlarge" ]]; then
+            worker_type=m4.xlarge
+            if [[ "${CLUSTER_VARIANT}" == "xlarge_worker" ]]; then
+              worker_type=m5.8xlarge
+            elif [[ "${CLUSTER_VARIANT}" =~ "xlarge" ]]; then
```

Comment:
not quite the best of both worlds since you won't be able to get xlarge workers + xlarge masters at the same time if you want them.  admittedly no one should want that today since you're the first person needing anything like this, but it'll be something to iterate on in the future.

let's see how this does in rehearsals and see if @smarterclayton has any comments (i think he's likely the one who introduced the cluster_variant concept in the first place).